recipes/git-pull-request/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "git-pull-request" %}
+{% set version = "6.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6a14c0dc748653833eff3e1822e544f8b5ac45b79b015b643f8805b6d75fc590
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - git-pull-request = git_pull_request:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - attrs
```

I mean, you should add it here :grimacing: 
```suggestion
    - attrs
    - git
```

---

done

---

Question: can I already now add the `workflows` folder to `.github` and populate it with the `automerge.yml` and `webservices.yml` files ?

---

under the motto "better to ask for forgiveness than permission" I added them, see what happens :stuck_out_tongue_winking_eye:

---

checks pass, I presume it is OK like this.